src/OrbitGl/TimeGraph.cpp

```diff
@@ -564,6 +569,54 @@ void TimeGraph::ProcessCGroupAndProcessMemoryTrackingTimer(const TimerInfo& time
   }
 }
 
+void TimeGraph::ProcessPagefaultTrackingTimer(const orbit_client_protos::TimerInfo& timer_info) {
+  uint64_t cgroup_name_hash = timer_info.registers(
+      static_cast<size_t>(CaptureEventProcessor::PagefaultEncodingIndex::kCGroupNameHash));
+  std::string cgroup_name = app_->GetStringManager()->Get(cgroup_name_hash).value_or("");
+  int64_t system_pagefault = orbit_api::Decode<int64_t>(timer_info.registers(
```

Comment:
I would prefer system_pagefaults (with a 's') or num_system_pagefault

Reply:
Changed accordingly! Thanks for suggesting the name!